src/content/docs/en/contributing/codeofconduct.mdx

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
I get that we should have an extensive CoC, but we probably want a short "essence", since most people aren't going to read the full thing